pkg/plugin/app.go

```diff
@@ -59,68 +53,85 @@ func NewDashboardReporterApp(ctx context.Context, settings backend.AppInstanceSe
 	app.registerRoutes(mux)
 	app.CallResourceHandler = httpadapter.New(mux)
 
-	// Load plugin settings
-	config, secrets, err := loadSettings(settings.JSONData, settings.DecryptedSecureJSONData)
-	if err != nil {
-		return nil, fmt.Errorf("failed to create dashboard-reporter plugin app instance: %s", err)
-	}
-	ctxLogger.Info("provisioned config", "config", config.String())
-	if secrets.token != "" {
-		ctxLogger.Info("service account token configured")
-	}
-
 	// Get default HTTP client options
 	opts, err := settings.HTTPClientOptions(ctx)
 	if err != nil {
 		return nil, fmt.Errorf("error in http client options: %w", err)
 	}
 
-	// If skip verify is set to true configure it for Grafana HTTP client
-	if config.SkipTLSCheck {
-		opts.TLS = &httpclient.TLSOptions{InsecureSkipVerify: true}
+	if opts.TLS == nil {
+		opts.TLS = &httpclient.TLSOptions{}
+	}
+
+	// Only allow configuring using GF_* env vars
+	// TODO Deprecated: Use tlsSkipVerify instead
+	if os.Getenv("GF_REPORTER_PLUGIN_IGNORE_HTTPS_ERRORS") != "" {
+		opts.TLS.InsecureSkipVerify = true
```

Comment:
This is bit problematic, since skip TLS will not handle by `config.Load`. 

The skip TLS config is handled by Grafana SDK plugin it-self (https://github.com/grafana/grafana-plugin-sdk-go/blob/c8a47594ccdb281c69df4fbd74f7f233c64ab25b/backend/http_settings.go#L222), this env var is more or less a hack and i would completely remove it. That would make it easier